modules/miscutil/lib/mailutils.py

```diff
@@ -24,24 +24,24 @@
 
 __revision__ = "$Id$"
 
+import os
+import re
 import sys
+from cStringIO import StringIO
 from time import sleep
+
+import six
```

Comment:
Woops. `six` on master is probably unnecessary to begin with. Replaced with `basestring` (as `six` does: https://bitbucket.org/gutworth/six/src/78315705481c3f5e491a3816e41aa66c54c8f965/six.py?at=default#cl-49 )